src/systems/audio-system.js

```diff
@@ -1,3 +1,16 @@
+let delayedReconnectTimeout = null;
+function performDelayedReconnect(gainNode) {
+  if (delayedReconnectTimeout) {
+    clearTimeout(delayedReconnectTimeout);
+  }
+
+  delayedReconnectTimeout = setTimeout(() => {
```

Comment:
Now we do retries to reconnect for both signaling and ICE after disconnects/failures so to keep this aligned with that behavior maybe it would make sense to do a `setInterval` here instead of a `setTimeout` here too so we keep on retrying until it reconnects? Also keeping the retry time aligned with the other retry times of 2secs might be good so all services go up at the same time.

Reply:
Can you also make sure  that the linter is working so the code structure is aligned with the rest of the app?

Reply:
It's actually a local RTCPeerConnection and from my tests it doesn't seem to be an issue if you lose the connection to the server. But it goes into disconnected state if network changed. In my tests I power off the wired network to force the switch to Wi-Fi so probably the wired network interface is gone. So the setTimeout seems enough in this particular case.
You can test this further if you want, but I'm not gonna spend any more time on it.